Sources/XMTPiOS/Conversations.swift

```diff
@@ -48,6 +63,18 @@ public actor Conversations {
 		return try await v3Client.conversations().list(opts: options).map { $0.fromFFI(client: client) }
 	}
 
+	public func streamGroups() async throws -> AsyncThrowingStream<Group, Error> {
```

Comment:
I'm not sure how you're going to stream both groups and conversations at the same time. But to be able to do this in Android the stream had to return the same type.

Reply:
Currently planning on holding off on having a combined stream until we do the Conversation -> DM rename. Then we can just have that stream return the union.